components/GraphDiagram/GraphDiagram.tsx

```diff
@@ -89,14 +89,14 @@ export default function GraphDiagram({ activity }: { activity: LoadActivity }) {
   // Signal for when Graph DOM changes
   const [domSignal, setDomSignal] = useState(0);
 
-  async function handleGraphClick(event: React.MouseEvent) {
-    const target = event.target as HTMLDivElement;
+  async function handleGraphClick(event: React.MouseEvent<HTMLElement>) {
+    const target = event.currentTarget;
 
-    if ($('#graph-controls').contains(target)) return;
+    if ($('#graph-controls')!.contains(target)) return;
 
-    const el = $.up<SVGElement>(target, '.node');
+    const el = target.closest('g.node');
 
-    const moduleKey = el ? $(el, 'title')?.textContent?.trim() : '';
+    const moduleKey = el ? $('title', el)?.textContent?.trim() : '';
```

Comment:
Note that select-dom, like the console API, takes the parent as the second parameter.